google-cloud-spanner-hibernate-samples/pom.xml

```diff
@@ -5,7 +5,7 @@
 	<parent>
 		<artifactId>google-cloud-spanner-hibernate</artifactId>
 		<groupId>com.google.cloud</groupId>
-		<version>1.5.4</version>
+		<version>1.5.4</version><!-- {x-version-update:google-cloud-spanner-hibernate:current} -->
```

Comment:
should this one have `-samples` suffix?

Reply:
Discussed offline; all versions will be synchronized on a single `google-cloud-spanner-hibernate:current` version in the release infrastructure comment.